.env.template

```diff
@@ -21,6 +21,9 @@
 ## EXIT_KEY - Key to exit AUTO-GPT
 # EXIT_KEY=n
 
+## SPIN - Enable spinner (happens when something in the background is running) (Default: True)
```

Comment:
Probably should make this a bit more generic for reduced output or just reuse CI envar

Reply:
What about renaming to SHOW_PROGRESS?

Reply:
I think `SHOW_PROGRESS_SPINNER` would be a good descriptive name. Would be good to include a comment along the lines of `Show the spinning progress symbol while 'thinking'. This can clutter standard output when not in an interactive terminal`

Reply:
How about PLAIN_OUTPUT?

I explicitly do not want something spinner specific

Reply:
I think `PLAIN_OUTPUT` with a comment mentioning that it disables the spinner is actually better than what I had suggested

Reply:
Lets swap all references to spin with cfg.plain_output and PLAIN_OUTPUT then?